client/src/components/Workflow/WorkflowInvocationsCount.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faClock, faSitemap } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { BBadge } from "bootstrap-vue";
+import { formatDistanceToNow, parseISO } from "date-fns";
+import { useRouter } from "vue-router/composables";
+
+import localize from "@/utils/localization";
+
+import UtcDate from "@/components/UtcDate.vue";
+
+library.add(faClock, faSitemap);
+
+interface Props {
+    workflow: any;
+}
+
+const props = defineProps<Props>();
+
+const router = useRouter();
+
+function onInvocations() {
+    router.push(`/workflows/${props.workflow.id}/invocations`);
+}
+</script>
+
+<template>
+    <div class="workflow-invocations-count d-flex align-items-center flex-gapx-1">
+        <BBadge
+            v-b-tooltip.hover
+            pill
+            :title="localize('View workflow invocations')"
+            class="outline-badge cursor-pointer list-view"
+            @click="onInvocations">
+            <FontAwesomeIcon :icon="faSitemap" />
+            <span v-if="workflow.run_count > 0">
+                workflow runs:
+                {{ workflow.run_count }}
+            </span>
+            <span v-else> workflow never run </span>
```

Comment:
Can we drop the run counts ? It doesn't seem to work (all my workflow are never run, which can't be right), but it's likely a very heavy request if we did that. This needs separate evaluation.